prqlc/prqlc/tests/integration/dbs/README.md

```diff
@@ -19,23 +19,22 @@ ClickHouse and GlareDB are tested — we use `docker compose`:
 1. Run `docker compose up` (may take a while on the first time):
 
    ```sh
-   cd prqlc/prqlc/tests/integration
-   docker compose up
+   cd prqlc/prqlc/tests/integration/dbs && docker compose up -d
    ```
 
 2. Run the tests:
 
    ```sh
-   cargo test --features=test-dbs-external --no-capture
+   cargo test --features=test-dbs-external -- --nocapture
    ```
 
-   The `--no-capture` option isn't required, but shows all the dialects tested
-   per query.
+   (The `--no-capture` option isn't required, but shows all the dialects tested
+   per query.)
 
 3. After it's done, stop the containers and remove local images and volumes:
 
    ```sh
-   docker compose down -v --rmi local
+   cd prqlc/prqlc/tests/integration/dbs && docker compose down
```

Comment:
Why remove options `-v --rmi local` ?

Reply:
The existing code removes the images, which makes iterating much longer... Is there an alternative?

Reply:
In that case, there is no problem.
I just thought it was safer for beginners to delete the volumes because in my experience it was annoying that official images such as postgres created volumes on their own and kept leaving them behind.

Reply:
We dont' have volumes anyway, `-v`/`--volumes` does nothing here.